components/CurrentBuildingLocation.js

```diff
@@ -27,8 +27,7 @@ function CurrentBuildingLocation () {
     // let currentFloor = 0;
 
     const _getLocationAsync = async () => {
-        let location = await Location.getCurrentPositionAsync({/*insert acuracy*/ });
-
+               let location = await Location.getCurrentPositionAsync({/*insert acuracy*/ });
```

Comment:
why are we using two different function to fetch the current location?

Reply:
I did not do any CurrentLocation outside of the scope of outdoor directions. Any other implementation wasn't done by me. We should definitely do one only one.